features/step_definitions/edition_update_slug_steps.rb

```diff
@@ -8,10 +8,10 @@
   click_button "Update"
 end
 
-Then(/^I can see the slug has been updated to "([^"]*)"$/) do |new_slug|
+Then(/^I can see the editions public url contains "([^"]*)"$/) do |new_slug|
   visit admin_edition_path(@edition)
   click_button "Create new edition"
-  expect(find("#edition_slug").value).to eq new_slug
+  expect(page).to have_content new_slug
```

Comment:
This matcher feels slightly misleading, because it would in theory match against anything on the page now – rather than directly in the `#edition_slug` element. I don't have an immediate solution. Maybe this is good enough tbh.

Another option could be to match against the entire page URL? e.g. something like:

```
I can see the edition's public URL contains "/government/edition-type/new-slug"
```

Which feels less likely to clash with anything else on the page.

No change needed necessarily – just something that I noticed.